core/src/com/mygdx/insanerpg/model/item/stackable/consumable/gem/testGem.java

```diff
@@ -0,0 +1,21 @@
+package com.mygdx.insanerpg.model.item.stackable.consumable.gem;
+
+public class testGem {
```

Comment:
Write a todo here that says something like this will not be in the final project, just for test

Reply:
added todo